### ISSUE TITLE
Issue when duplicating a page

### ISSUE THREAD

**Issue by rsmclaren** [sheadawson/silverstripe-linkable#74]
If you duplicate a page, and then modify the link (change Title, change link etc) those changes will be reflected on the original page (and any other duplicates). This is probably very similar to #52 

I've implemented a temporary workaround, for example in Page.php
````
public function onBeforeDuplicate(){
        parent::onBeforeDuplicate();

        if($this->LinkID >= 0){
            $this->LinkID = 0;
        }
    }
````

**Comment by sheadawson** [comment]
Good spotting @rsmclaren. I think what we need to do here is add a DataExtension to all DataObjects that looks for has_one Link relations onBeforeDuplicate. Maybe with a config setting that allows developers to specify the behaviour - whether to duplicate the Link objects or just remove the relationship you have done in your example

**Comment by rsmclaren** [comment]
That's a great idea. I'll try to set something up soon and do a PR.

**Comment by rsmclaren** [comment]
This was fixed in #75 - it would be great to get a new release with this fix included.

**Comment by sheadawson** [comment]
1.3.6 released :)
